theano/tensor/elemwise.py

```diff
@@ -125,6 +125,7 @@ class DimShuffle(Op):
 
     _f16_ok = True
     check_input = False
+    __props__ = ("input_broadcastable", "new_order")
```

Comment:
No problem. Take care of you. At the same time, this PR need a rebase when you have time to look at it again.